mocks/firebase.js

```diff
@@ -21,10 +21,19 @@ const firebaseStub = overrides => {
 };
 
 const mockFirebase = (overrides = {}) => {
-  jest.mock('firebase', () => firebaseStub(overrides)) &&
-    jest.mock('firebase-admin', () => firebaseStub(overrides));
+  mockModuleIfFound('firebase', overrides);
+  mockModuleIfFound('firebase-admin', overrides);
 };
 
+function mockModuleIfFound(moduleName, overrides) {
+  try {
+    require.resolve(moduleName);
+    jest.doMock(moduleName, () => firebaseStub(overrides));
+  } catch (e) {
+    // module ${moduleName} not found, skipping
```

Comment:
Could you add a `console.error` so there is at least something meaningful logged?

Reply:
I've added `console.info`, as it's not necessarily an error. Also, I had to disable eslint check for this line, hope this is acceptable.